setup.ps1

```diff
@@ -135,14 +135,39 @@ Write-Host "All software installation tasks completed. Press Enter to continue..
 
 $overwrite = Read-Host "If you use the VA VPN, public certs are required to interact with certain resources. Download and install? (Y/N)"
 if ($overwrite -eq 'Y') {
-    Write-Host "Downloading VA Cert..."
-    Invoke-WebRequest -Uri "https://department-of-veterans-affairs.github.io/github-handbook/guides/security/VA-Internal-S2-RCA1-v1.pem" -OutFile "$targetDir\VA-Internal-S2-RCA1-v1.pem"
-    Write-Host "Adding CA cert to cacerts files..."
-    # Find all directories containing the specified file
-    $cacertsFiles = Get-ChildItem -Path $targetDir -Recurse -Filter "cacerts" | Select-Object -ExpandProperty DirectoryName -Unique
-    foreach ($cacertFile in $cacertsFiles) {
-        Write-Host "Adding cert to $cacertFile"
-        keytool.exe -import -alias va-certificate-authority -file "$targetDir\VA-Internal-S2-RCA1-v1.pem" -keystore $cacertFile -storepass changeit -noprompt
+    Write-Host "Downloading Cacerts..."
+    Invoke-WebRequest -Uri "https://github.com/department-of-veterans-affairs/hats-cucumber-framework-setup/blob/main/cacerts" -OutFile "$targetDir\cacerts"
+    
+    # Check if the cacerts file was downloaded
+    if (-Not (Test-Path -Path "$targetDir\cacerts")) {
```

Comment:
In what situation would cacerts not exist here _and_ a previous error wasn't thrown in 139?

Reply:
One scenario where cacerts might not exist after download attempt without throwing error at 139 - network issues e.g. temporary loss of internet connectivity that occurs after Invoke-WebRequest command is executed but before the file is fully downloaded, the command may complete without error, but the file may not be saved as expected.

Reply:
removed